src/main/java/com/exasol/adapter/document/UdfCountCalculator.java

```diff
@@ -0,0 +1,48 @@
+package com.exasol.adapter.document;
```

Comment:
This is nice. We could use this in other UDF apps. I think we should start thinking on common utilities library.